Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -133,21 +133,30 @@ public NodejsProjectNode(NodejsProjectPackage package)
             }
         }
 
-        private void TryToAcquireTypings() {
+        private void TryToAcquireTypings(IEnumerable<string> packages) {
+            if (ShouldAcquireTypingsAutomatically && TypingsAcquirer != null) {
+                TypingsAcquirer
+                    .AcquireTypings(packages, null /*redirector*/)
+                    .ContinueWith(x => x);
+            }
+        }
+
+        private void TryToAcquireCurrentTypings() {
             var controller = ModulesNode != null ? ModulesNode.NpmController : null;
-            if (!ShouldAcquireTypingsAutomatically || TypingsAcquirer == null || controller == null) {
+            if (controller == null) {
                 return;
             }
 
-            var currentPackages = controller.RootPackage.Modules.Where(package =>
-                package.IsDependency
-                || package.IsOptionalDependency
-                || package.IsDevDependency
-                || !package.IsListedInParentPackageJson);
+            var currentPackages = controller.RootPackage.Modules
+                .Where(package =>
+                    package.IsDependency
+                    || package.IsOptionalDependency
+                    || package.IsDevDependency
+                    || !package.IsListedInParentPackageJson)
+                .Select(package => package.Name)
+                .Concat(new[] { "node" });
```

Comment:
I don't think you need to include "node" here at all anymore since Reload will always get called first.